demo/client/main.js

```diff
@@ -1,14 +1,28 @@
 import { ProseMeteorEditor } from 'meteor/prosemeteor:prosemirror';
 import { Meteor } from 'meteor/meteor';
 
-const docActivityTimeoutCallback = ({ docId }) => {
-  alert(`Document ${docId} activity timeout reached, shutting down server session`);
+const docActivityTimeoutCallback = ({ docId, elementId }) => {
+  document.getElementById(elementId).innerHTML =
+   `<div style="background-color: #fff9a4;text-align:center;vertical-align:center;"> 
+       Document ${docId} activity timeout reached, server session was shut down.
+    </div>`;
+};
+const authenticationErrorCallback = ({ userId, docId, type, elementId }) => {
+  document.getElementById(elementId).innerHTML =
+  `<div style="background-color: #ffa4a4;text-align:center;vertical-align:center;"> 
+        Authentication Error: user id "${userId}" isn\'t allowed to "${type}" document "${docId}".
+    </div>`;
 };
 
 Meteor.startup(function() {
-  let editor1 = new ProseMeteorEditor({
+  const editor1 = new ProseMeteorEditor({
     docId: 'proofOfConceptDocId1',
-    onDocActivityTimeout: docActivityTimeoutCallback,
+    onDocActivityTimeout: ({ docId }) => {
+      docActivityTimeoutCallback({ docId, elementId: 'prosemeteor-poc-1' })
+    },
+    onAuthenticationError: ({ userId, type, docId }) => {
```

Comment:
`onAuthenticationError()` is handled on a per-doc basis on the client by providing a callback to the config object in the `ProseMeteorEditor` constructor